### PR TITLE
BG-17966 Add XTZ support

### PR DESCRIPTION
**Pull request by argjv**
Update account-lib to 1.0.2 and add Tezos controller.

### CODE REVIEW

File: modules/core/package.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
     "gen-docs": "typedoc"
   },
   "dependencies": {
-    "@bitgo/account-lib": "^1.0.0",
+    "@bitgo/account-lib": "https://github.com/BitGo/bitgo-account-lib#9bcafd9f213077de434ca79d2e489880a153b3eb",
```
Comment (argjv) [inline modules/core/package.json]: Waiting for new `account-lib` release

File: modules/core/src/v2/coins/xtz.ts [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+  }
+
+  /**
+   * Sign message with private key
+   *
+   * @param key
+   * @param message
+   */
+  signMessage(key: KeyPair, message: string | Buffer): Buffer {
+    throw new Error('Method not implemented');
```
Comment (tylerlevine) [inline modules/core/src/v2/coins/xtz.ts]: Use existing error type: https://github.com/BitGo/BitGoJS/blob/master/modules/core/src/errors.ts#L101

File: modules/core/src/v2/coins/xtz.ts [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+   * Builds a funds recovery transaction without BitGo.
+   * We need to do three queries during this:
+   * 1) Node query - how much money is in the account
+   * 2) Build transaction - build our transaction for the amount
+   * 3) Send signed build - send our signed build to a public node
+   * @param params
+   * @param callback
+   */
+  recover(params: any, callback?: NodeCallback<any>): Bluebird<any> {
+    throw new Error('Method not implemented');
```
Comment (tylerlevine) [inline modules/core/src/v2/coins/xtz.ts]: use existing error type

File: modules/core/src/v2/coins/txtz.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
+/**
```
Comment (tylerlevine) [inline modules/core/src/v2/coins/txtz.ts]: Add prettier annotation

**Pull request merged**